openupgrade_scripts/scripts/stock/16.0.1.1/post-migration.py

```diff
@@ -0,0 +1,6 @@
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.load_data(env.cr, "stock", "noupdate_changes.xml")
```

Comment:
and also this. 😉

```suggestion
    openupgrade.load_data(env.cr, "stock", "16.0.1.1/noupdate_changes.xml")
```

Reply:
@huguesdk Thank you, I missed this information